### ISSUE TITLE
Start-Service : Can't start service “Microsoft Service Fabric Host Service (FabricHostSvc)”

### ISSUE THREAD

**Issue by dream-y** [Azure/service-fabric-issues#341]
When i run the service fabric application with vs2015, it show the below error:

2>已开始执行脚本“Set-LocalClusterReady”。
2>Import-Module 'C:\Program Files\Microsoft SDKs\Service Fabric\Tools\Scripts\DefaultLocalClusterSetup.psm1'; Set-LocalClusterReady
2>--------------------------------------------
2>Local Service Fabric Cluster is not running...
2>Please wait while we start the Local Service Fabric Cluster. This may take few minutes...
2>Starting service FabricHostSvc. This may take a few minutes...
2>**Start-Service : Can't start service “Microsoft Service Fabric Host Service (FabricHostS
2>vc)”。**
2>所在位置 C:\Program Files\Microsoft SDKs\Service Fabric\Tools\Scripts\ClusterSe
2>tupUtilities.psm1:458 字符: 5
2>+     Start-Service FabricHostSvc -WarningAction SilentlyContinue
2>+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2>    + CategoryInfo          : OpenError: (System.ServiceProcess.ServiceControl 
2>   ler:ServiceController) [Start-Service], ServiceCommandException
2>    + FullyQualifiedErrorId : StartServiceFailed,Microsoft.PowerShell.Commands 
2>   .StartServiceCommand
2> 
2>StartLocalCluster : Could not start FabricHostSvc
2>所在位置 C:\Program Files\Microsoft SDKs\Service Fabric\Tools\Scripts\DefaultLo
2>calClusterSetup.psm1:52 字符: 9
2>+         StartLocalCluster
2>+         ~~~~~~~~~~~~~~~~~
2>    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorExcep 
2>   tion
2>    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorExceptio 
2>   n,StartLocalCluster
2> 
2>ScriptHalted
2>所在位置 C:\Program Files\Microsoft SDKs\Service Fabric\Tools\Scripts\ClusterSe
2>tupUtilities.psm1:462 字符: 9
2>+         throw
2>+         ~~~~~
2>    + CategoryInfo          : OperationStopped: (:) [], RuntimeException
2>    + FullyQualifiedErrorId : ScriptHalted
2> 
2>脚本“Set-LocalClusterReady”已执行完毕。
2>所用时间: 00:05:05.2456157
2>PowerShell 脚本未能执行。

And, i run the command "Start-Service FabricHostSvc -WarningAction SilentlyContinue" in ps,
the same error appears.

Any advice is appreciated. Thanks.

**Comment by MikkelHegn** [comment]
@rishirsinha

**Comment by rishirsinha** [comment]
Can you please share the traces from the sfroot\logs folder for investigation?

**Comment by dream-y** [comment]
I can't find SfDevCluster folder, this is the output screenshot:
![Uploading Fabric.png…]()

**Comment by rishirsinha** [comment]
Can you check to see if this folder exists - C:\SfDevCluster\Log?

If not please run the following:
reg query "HKEY_LOCAL_MACHINE\Software\Microsoft\Service Fabric"

Look at the value of FabricLogRoot and send the contents of that folder.

**Comment by rishirsinha** [comment]
It looks like firewall manipulation is failing on this machine. What locale are you running in?

**Comment by dream-y** [comment]
It running in vs2015 with windows 7.

**Comment by rishirsinha** [comment]
Is it EN-US locale? If not what is the locale?

**Comment by dream-y** [comment]
I find that the question was caused by Windows Firewall.
When i open the windows firewall service, it's ok.

Thanks a million!

**Comment by dream-y** [comment]
And  my local is CN.

**Comment by muyiwa** [comment]
Thanks!!Working nw

**Comment by NTurakulov** [comment]
I'm facing problem that looks similar to the topic starter: can'r start Service Fabric Host Service.
Windows 10 (locale ru-RU), Visual Studio 2017, Service Fabric 6.3.187/ SDK 3.2.187

I've tried:
- restart Service Fabric
- repair/reinstall SF
- Unregister-ScheduledTask FabricCounters  

nothing helped. Trying to start fabrichost.exe from cmd I receive next exception message:
> Unhandled exception. (exception class std::system_error Common::ServiceBase::Run:159: StartServiceCtrlDispatcherW(ServiceTable): ╧ЁюЎхёё ёыєцс√ эх ьюцхЄ єёЄрэютшЄ№ ёт ч№ ё ъюэЄЁюыыхЁюь ёыєцс√. )

As it can be seen - error message is shown with wrong encodig.
@rishirsinha why did you ask about locale, does it matter? I've installed Service Fabric on my other machines and never has faced such issue
